problems/1.types+operators.js

```diff
@@ -2,15 +2,21 @@
 * @param {any} val - The input value
 * @returns {boolean} - Is val a number
 */
-function isNumber() {
-
+const isNumber = (val) => {
+  return typeof val === "number"
 }
 
 /** Takes in a value and returns whether it is truthy
 * @param {any} val - The input value
 * @returns {boolean} - Is val truthy
 */
-function isTruthy() {
+const isTruthy = (val) => {
+  if (val = !val){
```

Comment:
Prefer to check for truthyness instead of using !. 
if(val) 

YOu're also doing assignment on line 14